.gitlab-ci.yml

```diff
@@ -1271,6 +1272,26 @@ sign-multiarch-manifest:
   before_script:
     - mv tags_to_sign_${MANIFEST} tags_to_sign
 
+xray-scan-docker:
+  only:
+    - main
+  except:
+    - schedules
+  extends: .container-scan
+  stage: xray-scan
+  parallel:
+    matrix:
+      # - CHILD: ["amd64", "arm64", "ppc64le", "2019", "2022"]
+      - CHILD: ["amd64"]  # the .container-scan template currently only supports linux/amd64 images
+  needs:
+    - push-multiarch-manifest
+    - sign-multiarch-manifest
+  variables:
+    create_jira: "true"
```

Comment:
will the created jira be auto-assigned to our project?

Reply:
> will the created jira be auto-assigned to our project?

The jira creation is handled by prodsec's template, but I believe this will be similar to the jira's created by the other scan jobs.